### ISSUE TITLE
Crash selecting refactoring

### ISSUE THREAD

**Issue by Debbus72** [JosefPihrt/Roslynator#357]
Just got a crash in Visual Studio 2017 (15.7.1). 
Have Roslynator 1.8.2 installed. 

I have a new property from which I want to return a new instance. However, I forgot to type in the "return" keyword, so the property (getter at least) is in error. 

It looks like this: 
```
      public object Test
      {
         get { new object();  }
      }
```

If you have this code in VS it will show a red scribbly under "get". 
Now, in reality I didn't have a new object, but I needed an instance from a class in an assembly that wasn't referenced yet. But the above code also triggers  the crash. 
I typed CTRL-. to bring up the popup-refactor menu. 
Using arrow-keys or mouse, when you hover over the menu-option: "Use expression-bodied member" the plug-in crashes in VS with the following stack-trace: 
```
System.InvalidOperationException : Operation is not valid due to the current state of the object.
   at Roslynator.CSharp.Refactorings.UseExpressionBodiedMemberRefactoring.GetReturnExpressionOrThrowExpression(AccessorListSyntax accessorList)
   at Roslynator.CSharp.Refactorings.UseExpressionBodiedMemberRefactoring.GetNewNode(SyntaxNode node)
   at Roslynator.CSharp.Refactorings.UseExpressionBodiedMemberRefactoring.RefactorAsync(Document document,SyntaxNode node,CancellationToken cancellationToken)
   at Roslynator.CSharp.Refactorings.AccessorDeclarationRefactoring.<>c__DisplayClass0_1.<ComputeRefactorings>b__3(CancellationToken ct)
   at Microsoft.CodeAnalysis.CodeActions.CodeAction.DocumentChangeAction.GetChangedDocumentAsync(CancellationToken cancellationToken)
   at async Microsoft.CodeAnalysis.CodeActions.CodeAction.GetChangedSolutionAsync(<Unknown Parameters>)
   at async Microsoft.CodeAnalysis.CodeActions.CodeAction.ComputeOperationsAsync(<Unknown Parameters>)
   at async Microsoft.CodeAnalysis.CodeActions.CodeAction.ComputePreviewOperationsAsync(<Unknown Parameters>)
   at async Microsoft.CodeAnalysis.CodeActions.CodeAction.GetPreviewOperationsAsync(<Unknown Parameters>)
   at async Microsoft.CodeAnalysis.Editor.Implementation.Suggestions.SuggestedAction.GetPreviewResultAsync(<Unknown Parameters>)
   at async Microsoft.CodeAnalysis.Editor.Implementation.Suggestions.SuggestedActionWithNestedFlavors.<>c__DisplayClass11_0.<GetPreviewAsync>b__0(<Unknown Parameters>)
   at async Microsoft.CodeAnalysis.Extensions.IExtensionManagerExtensions.PerformFunctionAsync[T](<Unknown Parameters>)
```

So it seems that it already crashes on the preview-window. 
Now I know that the code is already in error, but the original VS refactor (IDE0025  C# Use expression body for properties) does work... 
But then again, that one assumes that I wanted to put an "return" in there, and it inserted it even though it changes the code (which a refactor shouldn't do). 

Basically, I don't like that the plugin crashed, but I also don't like the behavior of the VS-refactor.
